model/countries/PL/PL-parsing-rules.yaml

```diff
@@ -0,0 +1,132 @@
+regex_definitions:
+  kBuildingValueRe:  # Regex for type "building"
+    regex_concat:
+      parts:
+      - regex_fragment: \d+  # House number
+      # Optional suffix for building ids with a letter suffix, e.g. 12a.
+      - regex_fragment: '(?:'
+      # Single character suffix (e.g. "12 a" and "12a")
+      - regex_fragment: \s*[[:alpha:]]\b
+      - regex_fragment: ')?'
+      wrap_non_capture: false
+      
+  # Regular expression to match the prefixes that indicate a house number.
+  kStreetOptionalPrefixRe:
+    regex_fragment: '(?:(?:ulica|ul\.?|aleja|al\.?|plac|pl\.?|skwer|rondo|osiedle|boczna|bulwar|droga|rynek|szosa|zaulek)\s*)?'
+
+  # Regular expression to match separator of house/building number and unit/apartment number.
+  kHouseNumberAndUnitSeparator:
+    regex_fragment: \s*\/\s*
+
+capture_definitions:
+  ParseBuildingLocation:
+    capture:
+      output: building-location
+      parts:
+      - capture:
+          output: street
+          prefix: {regex_reference: kStreetOptionalPrefixRe}
+          parts: [{regex_reference: kMultipleLazyWordsRe}]
+      - separator: {regex_reference: kWhitespaceSeparator}
+      - capture:
+          output: building-and-unit
+          parts:
+            - capture:
+                output: building
+                parts: [ {regex_reference: kBuildingValueRe} ]
+            - separator: {regex_reference: kHouseNumberAndUnitSeparator} 
+            - capture:
+                output: unit
+                parts: [{regex_fragment: '\d+'}]
+                quantifier: MATCH_OPTIONAL
+
+
+parsing_definitions:
+  building-location:
+    decomposition: 
+      capture_reference: ParseBuildingLocation
+
+  street-address-alternative-1:
+    decomposition:
+      capture: 
+        output: street-address-alternative-1
+        parts:
+        - capture_reference: ParseBuildingLocation
+
+
+test_parsing_definitions:
```

Comment:
As indicated in https://www.grcdi.nl/gsb/poland.html, apartment numbers can also be expressed using `m` or `m.` as separator. Let's add support that as well.

Reply:
As discussed with @battre, since we decided to land changes ASAP, it was acceptable to take into consideration only one format.
But if we are not landing code today, I can add more extensive support for apartment (with tests) till Monday noon.

Reply:
I think that `m` and `m.` should not be a separator but `unit-type` and the captured value should be the `unit-name`. I can imagine that also in poland there are other unit types than apartments (e.g. office).

Reply:
If we want to include the apartment (and other unit) types, I'd suggest in such case we can also use as separator slash ('/') or space (' ') --> in this scenario, I can adjust the logic for Monday.